File: javascript/Easy/1672-richest-customer-wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
    let maxWealth = 0;
    for (account of accounts){
        let wealth = 0
        for (money of account) {
            wealth += money
        }
        maxWealth = Math.max(maxWealth, wealth)
    }
    return maxWealth
    
};

File: python/Easy/1672-richest-customer-wealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        max_wealth = 0
        for account in accounts:
            wealth = 0
            for money in account:
                wealth += money
            max_wealth = max(max_wealth, wealth)
        
        return max_wealth